app/frontend/components/pages/login/mnemonicAuth.tsx

```diff
@@ -139,10 +140,10 @@ class LoadByMnemonicSectionClass extends Component<Props> {
 export default connect(
   (state) => ({
     mnemonicInputValue: state.mnemonicInputValue,
+    mnemonicValidationError: state.mnemonicValidationError,
+    //
```

Comment:
//

Reply:
nope, it is there to avoid prettier collapsing things :-)